src/main/java/com/synopsys/integration/jira/common/model/response/IssueTypeResponseModel.java

```diff
@@ -91,4 +93,15 @@ public Integer getAvatarId() {
     public Map<String, JsonElement> getFields() {
         return fields;
     }
+
+    public Map<String, IssueCreatemetaFieldResponseModel> getTypedFields(Gson gson) {
```

Comment:
Do you need the `Gson` parameter? Doesn't `JiraResponseModel` have a gson member?

Reply:
unfortunately gson is null as this object is nested deep and the current implementation of parsing only set gson at the top level. This issue exists because of OAuth using google libraries therefore requiring us to not use the json transformer which normally sets the gson for all nested objects.